src/components/heading/heading.test.js

```diff
@@ -0,0 +1,150 @@
+  describe("should render Heading component", () => {
+    it.each(specialCharacters)(
+      "should check children as %s for Heading component",
+      (children) => {
+        CypressMountWithProviders(
+          <HeadingComponent> {children} </HeadingComponent>
+        );
+        headingPreview().contains(children);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check title as %s for Heading component",
+      (titleText) => {
+        CypressMountWithProviders(<HeadingComponent title={titleText} />);
+        headingTitle().should("have.text", titleText);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check titleId as %s for Heading component",
+      (titleId) => {
+        CypressMountWithProviders(<HeadingComponent titleId={titleId} />);
+        headingTitle().should("have.id", titleId);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check subheader as %s for Heading component",
+      (subheader) => {
+        CypressMountWithProviders(<HeadingComponent subheader={subheader} />);
+        subheaderPreview().should("have.text", subheader);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check subtitleId as %s for Heading component",
+      (subtitleId) => {
+        CypressMountWithProviders(<HeadingComponent subtitleId={subtitleId} />);
+        subheaderPreview().should("have.id", subtitleId);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check help text as %s for Heading component",
+      (helpText) => {
+        CypressMountWithProviders(<HeadingComponent help={helpText} />);
+        getComponent("help").realHover();
+        getDataElementByValue("tooltip")
+          .should("be.visible")
+          .contains(helpText);
+      }
+    );
+
+    it.each(testData)(
+      "should check %s as help link for Heading component",
+      (helpLink) => {
+        CypressMountWithProviders(<HeadingComponent helpLink={helpLink} />);
+
+        getComponent("help").should("have.attr", "href", helpLink);
+      }
+    );
+
+    it.each(testData)(
+      "should check %s as back link for Heading component",
+      (backLink) => {
+        CypressMountWithProviders(<HeadingComponent backLink={backLink} />);
+
+        getComponent("link").children().should("have.attr", "href", backLink);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check helpAriaLabel as %s for Heading component",
+      (ariaLabel) => {
+        CypressMountWithProviders(
+          <HeadingComponent help="This is a Title" helpAriaLabel={ariaLabel} />
+        );
+        getComponent("help").should("have.attr", "aria-label", ariaLabel);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check pill for Heading component",
+      (pillText) => {
+        CypressMountWithProviders(
+          <HeadingComponent pills={<Pill>{pillText}</Pill>} />
+        );
+        getComponent("pill").should("have.text", pillText);
```

Comment:
we already have it defined in file I sent you :) could we just import it from the `pill -> index.js` file ?